src/OrchardCore.Modules/OrchardCore.Title/Handlers/TitlePartHandler.cs

```diff
@@ -30,9 +30,32 @@ public class TitlePartHandler : ContentPartHandler<TitlePart>
             S = stringLocalizer;
         }
 
-        public override async Task UpdatedAsync(UpdateContentContext context, TitlePart part)
+        public override Task UpdatedAsync(UpdateContentContext context, TitlePart part)
```

Comment:
```suggestion
        public override async Task UpdatedAsync(UpdateContentContext context, TitlePart part) => await SetTitleAsync(part);
```

Reply:
This isn't needed and it was left out for a reason. The method returns a task, async and await are not needed since they are called by the consumer.